src/toady/cli.py

```
@@ -741,6 +805,210 @@ def resolve(ctx: click.Context, thread_id: str, undo: bool, pretty: bool) -> Non
         ctx.exit(1)
 
 
+def _handle_bulk_resolve(
```

Pylint (3.3.7)

[refactor] Too many local variables (25/15)

(R0914)